src/lib/modules/Document/DocumentConference.js

```diff
@@ -22,22 +22,12 @@ export class DocumentConference extends Component {
                 <Table.Cell>Dates</Table.Cell>
                 <Table.Cell>{conf.dates}</Table.Cell>
               </Table.Row>
-              <Table.Row>
```

Comment:
is it moved somewhere ? or just removed? If removed completely, then it is not entirely what was needed, since this is general case, it should be only removed from cds-ils UI